kibana-reports/public/components/main/report_definition_details/report_definition_details.tsx

```diff
@@ -336,10 +336,10 @@ export function ReportDefinitionDetails(props) {
       // TODO: need better display
       timePeriod: moment.duration(timeDuration).humanize(),
       fileFormat: reportFormat,
-      reportHeader: reportParams.core_params.hasOwnProperty('header')
+      reportHeader: reportParams.core_params.hasOwnProperty('header') && reportParams.core_params.header != ""
```

Comment:
just a note it's better to use `!==` in general so no implicit conversion is done, although in this case it doesn't seem to matter